packages/api/src/brevo-contact/brevo-contacts.service.ts

```diff
@@ -112,7 +112,7 @@ export class BrevoContactsService {
         let totalCount = 0;
         const targetGroupIds: number[] = [];
         const limit = 50;
-        const where = { isMainList: false, scope };
+        const where = { isMainList: false };
```

Comment:
Yes, removing is possible. The function is only used in the updateBrevoContact, where the contact is already checked for the scope, so that it's not neccessary here anymore.